frontend/src/metabase/admin/settings/components/widgets/ModelCachingScheduleWidget/CronExpressionInput.module.css

```diff
@@ -0,0 +1,54 @@
+.customScheduleLabel {
+  display: block;
+  color: var(--mb-color-text-medium);
+  font-size: 0.875rem;
+  font-weight: 400;
+  line-height: 1.5rem;
+  margin-bottom: 6px;
+}
+
+.errorMessage {
+  color: var(--mb-color-error);
+  margin-top: 4px;
+}
+
+.styledInput {
+  width: 100%;
+  border: 1px solid darken(var(--mb-color-border), 10%);
+}
+
+.inputContainer {
+  position: relative;
+}
+
+.infoIcon {
+  position: absolute;
+  right: 1rem;
+  top: 33%;
+  color: var(--mb-color-text-medium);
+}
```

Comment:
You could use a Mantine `TextInput` with a `rightSection` to reduce the custom CSS: https://mantine.dev/core/text-input/#left-and-right-sections

Reply:
Good call!